### PR TITLE
RANCHER-1557 Eureka UI implementation

### PR DESCRIPTION
**Pull request by eldiiar-duishenaliev**

### CODE REVIEW

File: pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!groovy
+package folioRancher.folioNamespaceTools.manageNamespace.createNamespaceFromBranch
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile:2]: Please avoid this addings from idea, this is not reqired change

File: vars/uiBuild.groovy [hunk 2]
```diff
@@ -78,3 +114,9 @@ static String getModuleVersion(String moduleName, boolean releaseVersion = false
     throw new RuntimeException("Unable to get ${moduleName} version. Url: ${registry.getURL()}. Status code: ${registry.getResponseCode()}.")
   }
 }
+
+@NonCPS
+static def make_tpl(String tpl, Map data) {
```
Comment (OHaimanov) [inline vars/uiBuild.groovy:119]: Not ideal solution, please mark it as method that should be refactored

**Review by OHaimanov — APPROVED** [review]

File: vars/uiBuild.groovy (hunk 2 above)
Comment (eldiiar-duishenaliev) [reply vars/uiBuild.groovy:120]: Thanks Boss, marked it.

File: pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile (hunk 1 above)
Comment (eldiiar-duishenaliev) [reply pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile]: Sure Boss, thanks for the suggestion!

**Pull request merged**